config/routes.rb

```diff
@@ -3,5 +3,10 @@
 
   devise_for :users
   root 'posts#index'
+  resources :users, only: %i[show update] do
+    member do
+      get 'profile'
+    end
+  end
```

Comment:
単数系のリソースを使って、`/profile` でプロフィールを表示したい。
```suggestion
  resource :profile, only: %i[show edit update]
```

編集は `/profile/edit` になるイメージ。

usersにネストするかどうか
ログイン中のみ有効になる（current_user が対象）ので、ネストしなくても「ログイン中のユーザーのprofile」というのがわかるので不要かなと。

Reply:
↑の変更を適用したら、profie_controller を新しくつくる必要がある

show,editのviewは `users/profile` のものを流用できるとおもう